src/explore-education-statistics-admin/src/pages/edit-release/ReleaseSetupEditPage.tsx

```diff
@@ -61,11 +62,11 @@ const ReleaseSetupEditPage = ({
   const [releaseTypes, setReleaseTypes] = useState<IdLabelPair[]>();
 
   useEffect(() => {
-    const release = DummyPublicationsData.getReleaseSetupDetails(releaseId);
-
-    setReleaseSetupDetails(release);
-    setTimePeriodCoverageGroups(DummyReferenceData.timePeriodCoverageGroups);
-    setReleaseTypes(DummyReferenceData.releaseTypeOptions);
+    service.getReleaseSetupDetails(releaseId).then(release => {
+      setReleaseSetupDetails(release);
+      setTimePeriodCoverageGroups(DummyReferenceData.timePeriodCoverageGroups);
+      setReleaseTypes(DummyReferenceData.releaseTypeOptions);
```

Comment:
This call will also exist shortly (when the pull request is in) /api/meta/releasetypes

Reply:
Cool thanks!